Description: Use the original fork for now.
Author: Utkarsh Gupta <utkarsh@debian.org>
Forwarded: https://github.com/zyedidia/micro/issues/1498
Last-Update: 2020-02-21

--- a/go.sum
+++ b/go.sum
@@ -48,8 +48,8 @@
 github.com/zyedidia/go-shellquote v0.0.0-20200613203517-eccd813c0655/go.mod h1:1sTqqO+kcYzZp43M5VsJe1tns9IzlSeC9jB6c2+o/5Y=
 github.com/zyedidia/highlight v0.0.0-20170330143449-201131ce5cf5 h1:Zs6mpwXvlqpF9zHl5XaN0p5V4J9XvP+WBuiuXyIgqvc=
 github.com/zyedidia/highlight v0.0.0-20170330143449-201131ce5cf5/go.mod h1:c1r+Ob9tUTPB0FKWO1+x+Hsc/zNa45WdGq7Y38Ybip0=
-github.com/zyedidia/json5 v0.0.0-20200102012142-2da050b1a98d h1:zmDMkh22zXOB7gz8jFaI4GpI7llsPgzm38/jG0UgxjE=
-github.com/zyedidia/json5 v0.0.0-20200102012142-2da050b1a98d/go.mod h1:NDJSTTYWivnza6zkRapeX2/LwhKPEMQ7bJxqgDVT78I=
+github.com/flynn/json5 v0.0.0-20200102012142-2da050b1a98d h1:zmDMkh22zXOB7gz8jFaI4GpI7llsPgzm38/jG0UgxjE=
+github.com/flynn/json5 v0.0.0-20200102012142-2da050b1a98d/go.mod h1:NDJSTTYWivnza6zkRapeX2/LwhKPEMQ7bJxqgDVT78I=
 github.com/zyedidia/poller v1.0.1 h1:Tt9S3AxAjXwWGNiC2TUdRJkQDZSzCBNVQ4xXiQ7440s=
 github.com/zyedidia/poller v1.0.1/go.mod h1:vZXJOHGDcuK08GXhF6IAY0ZFd2WcgOR5DOTp84Uk5eE=
 github.com/zyedidia/pty v2.0.0+incompatible h1:Ou5vXL6tvjst+RV8sUFISbuKDnUJPhnpygApMFGweqw=
--- a/internal/action/bindings.go
+++ b/internal/action/bindings.go
@@ -10,7 +10,7 @@
 	"strings"
 	"unicode"
 
-	"github.com/zyedidia/json5"
+	"github.com/flynn/json5"
 	"github.com/zyedidia/micro/v2/internal/config"
 	"github.com/zyedidia/micro/v2/internal/screen"
 	"github.com/zyedidia/tcell/v2"
--- a/internal/config/plugin_installer.go
+++ b/internal/config/plugin_installer.go
@@ -15,7 +15,7 @@
 
 	"github.com/blang/semver"
 	lua "github.com/yuin/gopher-lua"
-	"github.com/zyedidia/json5"
+	"github.com/flynn/json5"
 	ulua "github.com/zyedidia/micro/v2/internal/lua"
 	"github.com/zyedidia/micro/v2/internal/util"
 )
--- a/internal/config/plugin_installer_test.go
+++ b/internal/config/plugin_installer_test.go
@@ -5,7 +5,7 @@
 
 	"github.com/blang/semver"
 
-	"github.com/zyedidia/json5"
+	"github.com/flynn/json5"
 )
 
 func TestDependencyResolving(t *testing.T) {
--- a/internal/config/settings.go
+++ b/internal/config/settings.go
@@ -12,7 +12,7 @@
 	"strings"
 
 	"github.com/zyedidia/glob"
-	"github.com/zyedidia/json5"
+	"github.com/flynn/json5"
 	"github.com/zyedidia/micro/v2/internal/util"
 	"golang.org/x/text/encoding/htmlindex"
 )
--- a/go.mod
+++ b/go.mod
@@ -15,7 +15,7 @@
 	github.com/zyedidia/clipboard v1.0.3
 	github.com/zyedidia/glob v0.0.0-20170209203856-dd4023a66dc3
 	github.com/zyedidia/highlight v0.0.0-20170330143449-201131ce5cf5
-	github.com/zyedidia/json5 v0.0.0-20200102012142-2da050b1a98d
+	github.com/flynn/json5 v0.0.0-20200102012142-2da050b1a98d
 	github.com/zyedidia/pty v2.0.0+incompatible // indirect
 	github.com/zyedidia/tcell/v2 v2.0.2
 	github.com/zyedidia/terminal v0.0.0-20180726154117-533c623e2415
